upgrade.md

```diff
@@ -0,0 +1,52 @@
+# Upgrading Prodfiler host-agent
+
+The upgrade process varies highly depending on the type of installation.
+
+The Prodfiler UI project menu `... > Instructions` contains the project-specific data that _may_ be needed to perform an upgrade.
+
+![Instructions menu](./pictures/instructions.png)
+
+Find below a list of upgrade instructions for each supported platform.
+
+### Binary installation
+
+Use the commands provided in the UI in the `Instructions` menu, switching the `Binary` tab.
+
+### Docker container
+
+Use the commands provided in the UI in the `Instructions` menu, switching to the `Docker` tab.
+
+### OS package in .deb format
+
+Use the commands provided in the UI in the `Instructions` menu, switching to the `DEB Package` tab.
+
+### OS package in .rpm format
+
+* Download the latest stable release visiting [the stable release page](https://releases.prodfiler.com/stable/index.html).
+* run the upgrade without changing any configs
+        
+      sudo rpm -U pf-host-agent-*.rpm
```

Comment:
nit: For documentation I think it is nicer if the long version is used instead of the short cut.
```suggestion
      sudo rpm --upgrade pf-host-agent-*.rpm
```